packages/web-components/src/components/masthead/__stories__/README.stories.mdx

```diff
@@ -161,6 +161,40 @@ const l1Data = {
 document.getElementById('masthead').l1Data = l1Data;
 ```
 
+#### L1 CTA
+
+By default, the L1 CTA renders as a button link.
+
+```javascript
+const l1Data = {
+  platform: {...}
+  menuItems: [...],
+  actions: {
+    cta: {
+      title: 'Link text',
+      url: 'https://www.example.com',
+    },
+  },
+}
+```
+
+You can also set a `ctaType` to render as a button that will integrate with the IBM Contact Module application on supported domains. Available CTA types are defined by the `L1_CTA_TYPES` enum defined in [the services-store package](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/packages/services-store/src/types/translateAPI.ts)
```

Comment:
Looks like the `L1_CTA_TYPES` enum is (will be) defined in https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/packages/web-components/src/components/masthead/defs.ts , eg. [here in the diff](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/11597/files#diff-2a7fd41d10da0ad7b15ec1fa2e5e8ebb40cb5d5f301d3065d9f2407e50258002R63). Link needs to be updated.

Reply:
D'oh! Good catch.